dts/msm8916-samsung.dts

```diff
@@ -34,4 +34,15 @@
 			i2c-address = <0x14>;
 		};
 	};
+
+	j5nlte {
+	       model = "Samsung J5N LTE (EUR)";
+	       compatible = "qcom,msm8916-mtp", "qcom,msm8916", "qcom,mtp", "qcom,msm8916", "lk2nd,device";
```

Comment:
Most of these compatibles are not needed. It should look like for the other devices, e.g.
```suggestion
	       compatible = "samsung,j5nlte", "qcom,msm8916", "lk2nd,device";
```